index.js

```
@@ -3,50 +3,50 @@ const path = require('path')
 const fs = require('fs')
 
 const bot = new Client()
-const { token, volume, roomId, showSongName } = require('./config.json')
+const { token, volume, roomId, showSongName, allowedFormats } = require('./config.json')
 
-const getRandomTrack = () => playlist[~~(playlist.length * Math.random())]
+const getRandomTrack = () => {
+  const rand = ~~(playlist.length * Math.random())
+  return (rand !== prevTrack) ? prevTrack = rand : getRandomTrack()
+}
 
 let prevTrack = ''
 let playlist = {}
-
-const musicFormats = ['flac', 'mp3', 'wav']
+let voiceChannel = null
+let channelId = roomId
 
 const getPlaylist = () => 
   fs.readdir('./music/', (err, files) => {
     if (err) throw err
 
-    const musicFiles = files.filter(f => musicFormats.includes(f.split('.').pop()))
+    const musicFiles = files.filter(f => allowedFormats.includes(f.split('.').pop()))
     if (musicFiles.length <= 0) throw "Didn't get any music :c\nPut music files inside 'music' folder."
```

It should throw a `new Error()`, not string

---

Let it throw `new RegerenceError()`